AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/backgroundfilter/BackgroundFilterProcessor.swift

```diff
@@ -192,7 +192,7 @@ public class BackgroundFilterProcessor {
         attributes[kCVPixelBufferPixelFormatTypeKey] = NSNumber(value: Int(kCVPixelFormatType_32BGRA))
         attributes[kCVPixelBufferWidthKey] = NSNumber(value: newWidth)
         attributes[kCVPixelBufferHeightKey] = NSNumber(value: newHeight)
-        attributes[kCVPixelBufferIOSurfacePropertiesKey] = [:] as NSObject
+        attributes[kCVPixelBufferIOSurfacePropertiesKey] = [AnyHashable: Any]() as NSObject
```

Comment:
Have you tried to build the demo? there are also similar issue in demo.

Reply:
in CoreImageVideoProcessor.swift and MetalVideoProcessor.swift

Reply:
Yes, and I changed these 2 files as well.